src/configModel/configModel.vala

```diff
@@ -200,8 +200,47 @@ namespace SwayNotificationCenter {
         }
     }
 
+    public enum NotificationUrgencyEnum {
+        UNSET = -1,
+        LOW = 0,
+        NORMAL = 1,
+        CRITICAL = 2;
+
+        public static NotificationUrgencyEnum from_value (string value) {
```

Comment:
For some reason, I believe that this function is never called by the JSON deserialization process (I tried to run `value.ascii_down()` but it was not called). Same for the enum just above.

Reply:
The JSON parser checks this automatically. I created such methods for other enums for use in other methods